src/app/core/redux/app/types.ts

```diff
@@ -0,0 +1,18 @@
+export const SET_LOADING = "SET_LOADING";
```

Comment:
Change this enum. Here `typeof` is a string. The statement below doesn't add any value.

Reply:
Made changes.